### PR TITLE
AF-1787 - Integrate AppFormer.js on Business Central

### PR DESCRIPTION
**Pull request by tiagobento**
On this PR:

- Configuration of new AppFormer.js component Spaces Screen

Related PRs:
https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/936
https://github.com/kiegroup/appformer/pull/662
https://github.com/kiegroup/kie-wb-common/pull/2526
https://github.com/kiegroup/drools-wb/pull/1090
https://github.com/kiegroup/jbpm-wb/pull/1314
https://github.com/kiegroup/optaplanner-wb/pull/329
https://github.com/kiegroup/kie-wb-distributions/pull/893

### CODE REVIEW

**Comment by tiagobento** [comment]
Jenkins, please retest this.

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by tiagobento** [comment]
jenkins please retest this

File: jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-showcase/pom.xml [hunk 1]
```diff
@@ -547,6 +547,7 @@
               <compileSourcesArtifact>org.uberfire:uberfire-preferences-api</compileSourcesArtifact>
               <compileSourcesArtifact>org.uberfire:uberfire-preferences-client</compileSourcesArtifact>
               <compileSourcesArtifact>org.uberfire:uberfire-ssh-api</compileSourcesArtifact>
+              <compileSourcesArtifact>org.uberfire:appformer-js-bridge</compileSourcesArtifact>
```
Comment (cristianonicolai) [inline jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-showcase/pom.xml]: @tiagobento no need to add the dependency to _appformer-js-bridge_ in this showcase?

Comment (tiagobento) [reply jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-showcase/pom.xml]: Just added it. Thanks!

**Comment by tiagobento** [comment]
Jenkins, please retest this.

**Comment by tiagobento** [comment]
Jenkins, please retest this.

**Pull request merged**